src/mscorlib/shared/System/IO/PathInternal.cs

```diff
@@ -135,5 +140,13 @@ internal static void CheckSearchPattern(string searchPattern)
                 searchPattern = searchPattern.Substring(index + 2);
             }
         }
+
+        internal static string NormalizeDirectorySeparators(string path) => string.IsNullOrEmpty(path) ? path : NormalizeDirectorySeparatorsIfNeccessary(path) ?? path;
```

Comment:
This line is ~160 characters long. It is not an improvement to use expression bodies for methods that are this complex. This should be written old fashioned way.

Reply:
(I think expression bodies make sense as the line is less than about 100 characters long.)